modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -70,6 +76,8 @@ Endpoints are displayed in alphabetical order.
 
 Console components include API Console, API Console Documentation, API Console Instances, API Console Navigation, and API Console Request Panel. In Community Builder, you can add multiple console components to the same page. Changes to a console component are visible in all other console components on that page that display the same API. For example, selecting the *Summary* tab in an API Console results in *Summary* tab selection in all API Consoles on the same page that display the same API.
 
+Data shown from Anypoint Platform: API instances and documentation from the AML Modeling Framework (AMF). AML is the Anything Modeling Language.
```

Comment:
THis should say the Anypoint component known by the user/admin where the data comes from. Where do this info comes from, ultimately from the visible product set? is it from API Manager? Exchange? these are the component names we should use, as other things like AML or AMF are not visibile or understandable for the users.

Reply:
Confirmed with eng: information in the Console comes from Anypoint Exchange

Reply:
Updated.